Gemfile.prod.lock

```diff
@@ -36,7 +36,7 @@ GIT
 
 GIT
   remote: https://github.com/3scale/prawn.git
-  revision: 88aead0d97e230d08e9f51a18711d01317965bfe
+  revision: 6b18589fab63fabef420c0ccb546af431ad46404
```

Comment:
Are yous sure this change belongs to this commit?

Reply:
it is not 😄 will drop it